framework/PostProcessors/InterfacedPostProcessor.py

```diff
@@ -157,69 +187,66 @@ def collectOutput(self, finishedJob, output):
     evaluations = finishedJob.getEvaluation()
     if isinstance(evaluations, Runners.Error):
       self.raiseAnError(RuntimeError, "No available output to collect (run possibly not finished yet)")
-
     evaluation = evaluations[1]
+    output.addExpectedMeta(set(self.metaKeys))
```

Comment:
Doesn't this conflict potentially with the hard-coded metavars kept in the individual IPPs?

Reply:
the IPP should not generate metavars